layers/meta-balena-raspberrypi/conf/samples/local.conf.sample

```diff
@@ -37,6 +38,8 @@ KERNEL_IMAGETYPE:raspberrypi4-64="Image.gz"
 KERNEL_BOOTCMD:raspberrypi4-64 = "booti"
 KERNEL_IMAGETYPE:raspberrypi0-2w-64 = "Image.gz"
 KERNEL_BOOTCMD:raspberrypi0-2w-64 = "booti"
+KERNEL_IMAGETYPE:revpi-connect-s = "Image.gz"
```

Comment:
If you add https://github.com/balena-os/balena-raspberrypi/blob/master/layers/meta-balena-raspberrypi/conf/machine/raspberrypi4-unipi-neuron.conf#L8-L10 in your revpi4.inc lile I suggested above, then these changes here can be removed